web-ui/src/components/member_selector/MemberSelector.test.jsx

```diff
@@ -46,7 +46,6 @@ describe('MemberSelector', () => {
           title="Custom Title"
           outlined
           exportable
-          listHeight={300}
```

Comment:
This prop was removed since we no longer display a list of the selected members in this component.